### DISCUSSION TITLE
Plugin Development: hiding plugin navbar item on organizer level when disabled

### DISCUSSION THREAD

**Discussion by moritzlerch** [pretix/pretix#3447] [Q&A] [answered]
Hey, 

I'm currently building a plugin to extend pretix. It is displayed on organizer level. Disabling it now on event level doesn't hide it from the organizer. How would I get this behaviour with my custom plugin?

e.g. the built in banktransfer plugin also does this: https://github.com/pretix/pretix/blob/8f7220b574f07c523aa87750f3b84f9df80b17a8/src/pretix/plugins/banktransfer/signals.py#L80

When I now replace the plugin name and use the code over within my plugin I can't get this behaviour. Maybe it is because it isn't contained directly within the `request.organizer.events`?

Thanks in advance!

**Comment by pc-coholic** [comment]
Can you provide the whole block that you are using in your signals.py?

**Comment by moritzlerch** [comment]
Thanks for the quick reply!

That would be the code I would like to have this feature in: 

```python
@receiver(nav_organizer, dispatch_uid="manualseats_orga_nav")
def control_nav_orga_manualseats(sender, request=None, **kwargs):
    url = resolve(request.path_info)
    if not request.user.has_organizer_permission(request.organizer, "can_change_organizer_settings", request=request):
        return []
     if not request.organizer.events.filter(plugins__icontains='pretix.plugins.pretix_manualseats'):  # problematic line
        return []
    return [
        {
            "label": _("Seating Plans"),
            "url": reverse(
                "plugins:pretix_manualseats:index",
                kwargs={
                    "organizer": request.organizer.slug,
                },
            ),
            "active": (url.namespace == "plugins:pretix_manualseats"),
            "icon": seat_icon,
        },
    ]
```

**Reply by pc-coholic** [reply, marked as answer]
Unless you actually put your plugin directly into the pretix core, your plugin name is probably just `pretix_manualseats` or similar.

If in doubt, check your plugins apps.py or the plugins column of the events database table.

**Reply by moritzlerch** [reply]
Thanks a lot! I thought I had tried that before, but now it works :)
